opengrok-web/src/main/java/org/opengrok/web/api/v1/controller/ProjectsController.java

```diff
@@ -118,9 +118,11 @@ public Response addProject(String projectName) {
                     }
                 }
                 // deleted repository
-                for (RepositoryInfo repo : map.get(project)) {
-                    if (!repos.contains(repo)) {
-                        allrepos.remove(repo);
+                if (map.get(project) != null) {
```

Comment:
```suggestion
                if (map.containsKey(project)) {
```
I think this is nicer :)